include/inexor/vulkan-renderer/wrapper/instance.hpp

```diff
@@ -13,18 +13,10 @@ namespace inexor::vulkan_renderer::wrapper {
 /// @note The instantiation of this class must be synchronized externally.
 class Instance {
 protected:
-    VkInstance instance = VK_NULL_HANDLE;
-    AvailabilityChecksManager availability_checks;
+    VkInstance m_instance = VK_NULL_HANDLE;
```

Comment:
We should probably keep the vulkan handle initializations, just to avoid the clang-tidy warning. But I will change the style from `VkInstance m_instance = VK_NULL_HANDLE;` to `VkInstance m_instance{VK_NULL_HANDLE};`